src/search/documents/paper.py

```diff
@@ -67,17 +70,68 @@ def should_remove_from_index(self, obj):
 
         return False
 
-    # Used specifically for "autocomplete" style suggest feature
-    def prepare_title_suggest(self, instance):
+    # Used specifically for "autocomplete" style suggest feature.
+    # Includes a bunch of phrases the user may search by.
+    def prepare_suggestion_phrases(self, instance):
+        phrases = []
+
+        # Variation of title which may be searched by users
+        if instance.title:
+            phrases.append(instance.title)
+            phrases.extend(instance.title.split())
+
+        if instance.doi:
+            phrases.append(instance.doi)
+
+        # Variation of journal name which may be searched by users
+        if instance.external_source:
+            journal_words = instance.external_source.split(" ")
+            phrases.append(instance.external_source)
+            phrases.extend(journal_words)
+
+        # Variation of OpenAlex keywords which may be searched by users
+        try:
+            oa_data = instance.open_alex_raw_json
+            keywords = [keyword_obj["keyword"] for keyword_obj in oa_data["keywords"]]
+            joined_kewords = " ".join(keywords)
+
+            phrases.append(joined_kewords)
+            phrases.extend(keywords)
+
+        except:
+            pass
```

Comment:
Nite: Avoid bare exceptions:

```suggestion
        except Exceptions:
            pass
```

See: https://stackoverflow.com/questions/54948548/what-is-wrong-with-using-a-bare-except

Reply:
Interesting. Didn't know about bare excepts!